.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+test.db
+*cache*
+backend/.venv
```

Comment:
Zit hier nog een pep no white linen at end of file melding

Reply:
fixed